packages/blade/src/components/Box/BaseBox/getBaseBoxStyles.web.test.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  getBaseBoxStyles,
+  getDependencyProps,
+  getResponsiveValue,
+  getSpacingValue,
+  shouldAddBreakpoint,
+} from './getBaseBoxStyles';
+import type { BaseBoxProps } from './types';
+import paymentLightTheme from '~components/BladeProvider/__tests__/paymentLightTheme/paymentLightTheme';
+
```

Comment:
Some tests are in `getBaseBoxStyles.test.ts`.

Tests for `getAllProps` and `getMediaQueries` function were remaining since they are already covered in `getBaseBoxStyles` but I added few individual tests for them as well